packages/go/cypher/analyzer/analyzer.go

```diff
@@ -152,11 +185,13 @@ func (s *ComplexityMeasure) onNodePattern(node *model.NodePattern) {
 		}
 
 		// Track this node pattern to see if any subsequent expressions will narrow its kind matchers
-		s.nodeLookupKinds[node.Binding] = nodeLookupKinds
+		s.nodeLookupKinds[nodePatternBinding.Symbol] = nodeLookupKinds
 	}
+
+	return nil
 }
 
-func (s *ComplexityMeasure) onRelationshipPattern(node *model.RelationshipPattern) {
+func (s *ComplexityMeasure) onRelationshipPattern(stack *model.WalkStack, node *model.RelationshipPattern) error {
```

Comment:
I can't remember if it's a compiler warning or not but you may want to consider renaming "stack" to "_" since that param is unused.